app/controllers/projects_controller.rb

```diff
@@ -183,6 +183,13 @@ def modules
     redirect_to action: 'settings', id: @project, tab: 'modules'
   end
 
+  def custom_fields
+    @project.update_attributes(params[:project])
+    flash[:notice] = l(:notice_successful_update)
+
+    redirect_to action: 'settings', id: @project, tab: 'custom_fields'
```

Comment:
* The method does not enforce authorization. Please add the action to [the ones requiring authorization](https://github.com/basiszwo/openproject/blob/20841-custom-fields/app/controllers/projects_controller.rb#L39). 
* There are no specs for this action
* The action does allow the user to alter more than just setting the custom_fields of the project. I could e.g. change the name of the project using the method by manipulating the form. You should either rely on [permitted params](https://github.com/basiszwo/openproject/blob/dev/app/models/permitted_params.rb) or you could take the desired values out of the hash yourself. As long as the permissions are related, this is not a problem. But if we introduce a separate permission, the user will be allowed to do more than anticipated.
* There is no error handling in the method. If I manipulate the form to send a non existing ID e.g. -1, an ActiveRecord error will be thrown. (granted, this weakness also exists in the `modules` method above.